src/search/containers/search-results/search-results.component.ts

```diff
@@ -63,6 +66,45 @@ export class SearchResultsComponent implements OnInit, OnDestroy {
 
     // Hide spinner
     this.showSpinner = false;
+
+    // Update moreResultsToGo flag
+    this.moreResultsToGo = searchResult.resultInfo.moreResultsToGo;
+  }
+
+  public getPreviousResultsPage(): void {
+    // Decrement the start record
+    this.searchService.decrementStartRecord();
```

Comment:
Ah... I knew this would come up, @andywilkinshmcts! Funny thing is originally, I did think about whether `decrementStartRecord()` should return the new value but I guess I decided to have the function just do one thing. I'll change it 🙂.